docs/configuration.md

```diff
@@ -33,6 +33,10 @@ Gohan server configuration uses YAML format.
       initial_data:
           - type: "yaml"
             connection: "./etc/examples/heat_template.yaml"
+      # if legacy is true table names are based on schema plural,
+      # otherwise table names are created from schema id,
+      # default false
+      legacy: true
```

Comment:
I approve this change. However, we need to take care when upgrading gohan. Upgrade might break existing environment.